src/app/course/challenge/list-of-teams/list-of-teams.component.html

```diff
@@ -56,7 +59,7 @@ <h3 class="tui-text_h3 tui-col_stretch flex-grow">List of Teams</h3>
                                 <ng-container *ngIf="isInTeam(team); else notInTeam;">
                                     <a
                                         *ngIf=
-                                            "isInTeam(team) && !(event.is_open || event.is_closed)"
+                                            "isInTeam(team) && !event.is_not_available_yet"
```

Comment:
I think it should be
```suggestion
                                            "isInTeam(team) && event.is_not_available_yet"
```